### PR TITLE
BCI-2953: Added Report Format for Aptos

### PR DESCRIPTION
**Pull request by yongkangchia**
Parent Ticket: https://smartcontract-it.atlassian.net/browse/BCI-2953
Parent PR: https://github.com/smartcontractkit/chainlink/pull/13564

Context:
* Aptos Keystore Integration is blocked by this addition ([CI](https://github.com/smartcontractkit/chainlink/pull/13564/checks)) for LLO integration tests. The integration tests checks `chainlink-common` for report format based on chaintypes. 
* Therefore adding `ReportFormatAptos` is necessary 

Changes: 
* Adding `ReportFormatAptos` and String functions

core ref: 83b25926682729610f768aa93f43ce1063d4336b

### CODE REVIEW

**Review by jmank88 — APPROVED** [review]

**Comment by yongkangchia** [comment]
Will merge this in once Core PR gets approved 
https://github.com/smartcontractkit/chainlink/pull/13564

**Pull request merged**
